src/ci/docker/disabled/dist-armebv7r-none-eabihf/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  g++ \
+  make \
+  file \
+  curl \
+  ca-certificates \
+  python2.7 \
+  git \
+  cmake \
+  sudo \
+  xz-utils \
+  bzip2 \
+  libssl-dev \
+  pkg-config
+
+
+COPY scripts/sccache.sh /scripts/
+RUN sh /scripts/sccache.sh
+
+ENV BASE_URL=https://releases.linaro.org/components/toolchain/binaries/latest/armeb-eabi/
+ENV GCC_LINARO=gcc-linaro-7.2.1-2017.11-x86_64_armeb-eabi
+
+RUN curl -sL $BASE_URL/$GCC_LINARO.tar.xz | tar -xJ
+
+ENV PATH=$PATH:/$GCC_LINARO/bin
+
+ENV TARGET=armebv7r-none-eabihf
+
+ENV CC_armebv7r_none_eabihf=armeb-eabi-gcc \
+    CFLAGS_armebv7r_none_eabihf="-mthumb -march=armv7-r"
```

Comment:
To match this rust target, you want `-marm` instead of `-mthumb`.

Reply:
Cortex-R has ARM+Thumb instruction (dual mode). Wake-up in ARM mode and can use Thumb instr. So here `armv7-r` select ARM mode, `-mthumb` extend support for Thumb.

The same logic also in LLVM: the triple `armebv7r-none-eabihf` and the feature `+thumb2`.
[This](https://github.com/paoloteti/ti-hercules-bsp/blob/master/boards/Makefile.common) is a working Makefile that use `-mthumb -march=armv7-r` and [this](https://github.com/paoloteti/ti-hercules-bsp/blob/master/boards/ti_tms570/armebv7r-none-eabihf.json) a working JSON.

Reply:
Ahh ok. Default shall be Arm. I'll move to `-marm` removing `+thumb2` too. 
I'll test again on target (TI TMS570LS3137) in the afternoon. Thanks.